main.cpp

```diff
@@ -14,11 +14,47 @@ Create a branch named Part2
     you should be able to deduce the return type of those functions based on their usage in Person::run()
     You'll need to insert the Person struct from the video in the space below.
  */
+struct Person	
+{	
+	int age;
+	int height;
+	float hairLength;
+	float GPA;
+	unsigned int SATScore;
 
+    struct Foot				
+	{			
+        float distanceFromGround = 0 ;
+        float momentum = 0;
 
+    	int stepForward( float momentum, float distanceFromGround );		
+		float stepSize();		
+	};			
 
+    Foot leftFoot;
+    Foot rightFoot;
 
+    void run( int howFast, bool startWithLeftFoot );
+};
+
+
+int Person::Foot::stepForward( float momentumT0, float distanceFromGroundT0 )				
+{			
+    if ( momentumT0*distanceFromGroundT0 > 0 )
+    
```

Comment:
be consistent with your spacing of `( )`.  Look at the Readme.md coding standard.
- [ ] revise